aospy/test/test_calc_basic.py

```diff
@@ -160,6 +161,56 @@ def setUp(self):
         }
 
 
+test_params = {
```

Comment:
nit:  indentation is longer than the standard 4 spaces.

(C.f. comment above re: line-splitting: for me this particular case falls into a gray area of intermediate length, where it could work throwing everything onto as few lines as possible, but it's also long enough that what you've done also works.)